public/views/user/list.html

```diff
@@ -26,16 +26,10 @@
                     <td data-title="'ID' | translate" sortable="'id'" class="text-center">
                         {{ user.id }}
                     </td>
-                    <td data-title="'ACTION' | translate" class="text-center">
-                        <a href="" ui-sref="user.show({ userId: user.id })" class="btn btn-default btn-xs">
-                            <i class="fa fa-search"></i>
-                        </a>
-                        <a href="" ui-sref="user.edit({ userId: user.id })" class="btn btn-primary btn-xs">
-                            <i class="fa fa-pencil"></i>
-                        </a>
-                        <button user-delete-button ng-disabled="currentUserId === user.id"
```

Comment:
The ng-disabled for current user is missing:

`ng-disabled="currentUserId === user.id"`